packages/core/src/components/Button/button-inverse.example.html

```diff
@@ -4,5 +4,7 @@
   <button class="ds-c-button ds-c-button--primary">Button</button>
   <button class="ds-c-button ds-c-button--danger">Button</button>
   <button class="ds-c-button ds-c-button--success">Button</button>
-  <button disabled class="ds-c-button ds-c-button--disabled-inverse">Button</button>
+  <button disabled class="ds-c-button ds-c-button--disabled-inverse">
+    Button
+  </button>
```

Comment:
I wonder why this button is treated differently than the rest here?

Reply:
I think its the length of the classes. Prettier will add line breaks if the line is too long